ng-dev/release/publish/actions.ts

```diff
@@ -415,7 +415,10 @@ export abstract class ReleaseAction {
     const releaseNotesCompareTag = getReleaseTagForVersion(compareVersionForReleaseNotes);
 
     // Fetch the compare tag so that commits for the release notes can be determined.
-    this.git.run(['fetch', this.git.getRepoGitUrl(), `refs/tags/${releaseNotesCompareTag}`]);
+    // We forcibly override existing local tags that are named similar as we will fetch
+    // the correct tag for release notes comparison from the upstream remote.
+    this.git.run(['fetch', '--force', this.git.getRepoGitUrl(),
+        `refs/tags/${releaseNotesCompareTag}:refs/tags/${releaseNotesCompareTag}`]);
```

Comment:
Note that `tag <name>` would also work as refspec but I think it's better being explicit here. Quote from Git docs:

> tag <tag> means the same as refs/tags/<tag>:refs/tags/<tag>; it requests fetching everything up to the given tag.